src/classes/AccountDeDuper.cls

```diff
@@ -1,4 +1,4 @@
-public with sharing class AccountDeDuper {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.